subsys/bluetooth/common/bt_to_str.h

```diff
@@ -15,8 +15,3 @@ const char *bt_hex_real(const void *buf, size_t len);
 const char *bt_addr_str_real(const bt_addr_t *addr);
 const char *bt_addr_le_str_real(const bt_addr_le_t *addr);
 const char *bt_uuid_str_real(const struct bt_uuid *uuid);
-
-#define bt_hex(buf, len) bt_hex_real(buf, len)
-#define bt_addr_str(addr) bt_addr_str_real(addr)
-#define bt_addr_le_str(addr) bt_addr_le_str_real(addr)
-#define bt_uuid_str(uuid) bt_uuid_str_real(uuid)
```

Comment:
You should also rename the actual c-functions, i.e. remove the `_real` suffix - that has no meaning since we're not wrapping the functions anymore.

Reply:
Done.